android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -76,11 +81,8 @@ public void testEnableAndDisableCustomLogging() {
 
     //region File Logging Tests
     @Test
-    public void testFileLoggingLoggingLevels() {
-        final File path = new File(
-                context.getCacheDir().getAbsolutePath(),
-                "testFileLoggingLoggingLevels"
-        );
+    public void testFileLoggingLoggingLevels() throws Exception {
+        final File path = tempFolder.newFolder();
```

Comment:
Wondering if we need to clean the folder when the test is done?

Reply:
it will remove itself, when the test finishes, this is what I read from documentation. 

https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html

Reply:
Seems like variable needs to be public. 
```
com.couchbase.lite.LogTest > initializationError[test(AVD) - 4.4.2] [31mFAILED [0m
	org.junit.internal.runners.rules.ValidationError: The @Rule 'tempFolder' must be public.

	at org.junit.internal.runners.rules.RuleMemberValidator$MemberMustBePublic.validate(RuleMemberValidator.java:222)
```